lib/sequelize.d.ts

```diff
@@ -109,6 +109,53 @@ export interface ReplicationOptions {
 
 }
 
+/**
+ * Object that holds all operator symbols
+ */
+export declare var Op: {
```

Comment:
Is this the right file?

Reply:
don't you think so? its exported alongside the sequelize class